src/components/reusable/table/story-helpers/column-setting.ts

```diff
@@ -46,12 +46,14 @@ class StoryColumSetting extends LitElement {
     }
     .t-head {
       top: 72px;
+      z-index: 2;
```

Comment:
This is not needed, as I already added a z-index to the `stickyheader`.

Reply:
I saw but its not working for lock icon. lock icon still visible while hover on row and scroll.